components/DynamicComponent.jsx

```diff
@@ -0,0 +1,18 @@
+import dynamic from "next/dynamic";
+
+const components = {
+    node: dynamic(() => import('./Node')),
+    'node-article-full': dynamic(() => import('./NodeArticleFull')),
```

Comment:
can we keep compnent names 1:1 named like the custom-element, thus stay with kebap case? we should follow how the nuxt demo works unless not otherwise possible